### PR TITLE
[Dashboard] CPU/GPU usage details in actor pane

### PR DESCRIPTION
**Pull request by mfitton**
## Why are these changes needed?
This adds bars to see CPU and GPU utilization to each actor pane.
image

This image doesn't have any GPUs in it. I'll spin this up on a cluster later since I forgot to take a screenshot of the GPU utilization component.

## Checks

- [x] I've run `scripts/format.sh` to lint the changes in this PR.
- [x] I've included any doc changes needed for https://docs.ray.io/en/master/.
- [x] I've made sure the tests are passing. Note that there might be a few flaky tests, see the recent failures at https://flakey-tests.ray.io/
- Testing Strategy
   - [x] Unit tests
   - [ ] Release tests
   - [ ] This PR is not tested :(

### CODE REVIEW

**Comment by edoakes** [comment]
@mfitton Nice, this is awesome! At first glance the visual bar for CPU looks really big to me -- maybe shrink it to the size that shows in the machine view for consistency? I would also imagine this is probably some of the most important information so I'd suggest putting it above the summary stats (maybe to the right of the actor name?).

**Comment by mfitton** [comment]
@edoakes that makes sense, I'll collapse it so that both fit on one line rather than two lines, hoist it up above the details pane, and add a percent sign (since I just noticed that missing).

**Comment by mfitton** [comment]
Screen Shot 2020-10-12 at 9 35 30 PM

@edoakes here's the new look.

File: python/ray/setup-dev.py [hunk 1]
```diff
@@ -60,6 +60,7 @@ def do_link(package, force=False, local_path=""):
     do_link("experimental", force=args.yes)
     do_link("util", force=args.yes)
     do_link("dashboard", force=args.yes)
+    do_link("new_dashboard", force=args.yes)
```
Comment (edoakes) [inline python/ray/setup-dev.py:63]: shouldn't this be in the other PR?

File: dashboard/modules/reporter/reporter_head.py [hunk 2]
```diff
@@ -24,6 +24,7 @@ class ReportHead(dashboard_utils.DashboardHeadModule):
     def __init__(self, dashboard_head):
         super().__init__(dashboard_head)
         self._stubs = {}
+        self._ray_config = None
```
Comment (edoakes) [inline dashboard/modules/reporter/reporter_head.py:27]: intentional?

**Review by edoakes — APPROVED** [review]

File: python/ray/setup-dev.py (hunk 1 above)
Comment (mfitton) [reply python/ray/setup-dev.py:63]: I cherry-picked the commit to both for testing purposes.

File: dashboard/modules/reporter/reporter_head.py (hunk 2 above)
Comment (mfitton) [reply dashboard/modules/reporter/reporter_head.py:27]: Yep, this is a fix for a bug that I added during iteration on a PR from a previous PR.

File: python/ray/setup-dev.py (hunk 1 above)
Comment (mfitton) [reply python/ray/setup-dev.py:63]: I cherry-picked to both for testing.

**Pull request merged**